.flake8

```diff
@@ -0,0 +1,4 @@
+# Please do not modify the content of this file.
+[flake8]
+filename_check1 = filter_regex=test_.+
```

Comment:
what does this reinforce?

Reply:
Great question!.  I misunderstood how this tool worked.  I am going to close for now and then reopen when the tests and examples for this tool are in an operational state.